### PR TITLE
[silly] update config.guess and config.sub to support more platforms.

### LINKED ISSUE #68: [silly] Outdated config.guess and config.sub can not compile on aarch64
Here are newer config.guess and config.sub in order to be able to compile under aarch64 which can be added to the source array:

'config.guess::http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD'
'config.sub::http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD'

And add to the prepare something like:

cp ../config.guess .
cp ../config.sub .

### PR DESCRIPTION
**Pull request by jgmdev**
Fixes #68

### CODE REVIEW

File: silly/PKGBUILD [hunk 1]
```diff
@@ -4,24 +4,32 @@
 
 pkgname=silly
 pkgver=0.1.0
-pkgrel=8
+pkgrel=9
 pkgdesc="Simple Image Loading LibrarY, a part of the CEGUI project"
-arch=('x86_64')
+arch=('x86_64' 'aarch64')
```
Comment (acxz) [inline silly/PKGBUILD:9]:
```suggestion
arch=('i686' 'x86_64' 'aarch64' 'armv7h' 'armv6h')
```

Comment (acxz) [reply silly/PKGBUILD:9]: Let's have em all!

Comment (jgmdev) [reply silly/PKGBUILD]: Ahh! totally forgot about .SRCINFO updated it and added all the arch's

**Comment by acxz** [comment]
LGTM

**Pull request merged**